### ISSUE TITLE
bad-continuation is still present in 2.5.3

### ISSUE THREAD

**Issue by silpheel** [PyCQA/pylint#3761]
### Steps to reproduce
1. Prepare a python file with indentation in the signature as described in Current behavior.
2. run pylint
3. bad-continuation is reported

### Current behavior
```
> cat badcont.py
def test(
    a, b
):
    print(a, b)

> pylint badcont.py
************* Module badcont
badcont.py:2:0: C0330: Wrong hanging indentation before block (add 4 spaces).
    a, b
    ^   | (bad-continuation)
badcont.py:1:0: C0114: Missing module docstring (missing-module-docstring)
badcont.py:1:0: C0116: Missing function or method docstring (missing-function-docstring)

---------------------------------------------------------------------
Your code has been rated at -5.00/10 
```

### Expected behavior
bad-continuation should have been removed since 2.5.1 as per https://github.com/PyCQA/pylint/pull/3571

### pylint --version output
pylint 2.5.3
astroid 2.4.2
Python 3.6.11 (default, Jul 17 2020, 14:57:20)
[GCC 4.2.1 Compatible Apple LLVM 11.0.3 (clang-1103.0.32.29)]

**Comment by danrneal** [comment]
For some reason, the file downloaded from pypi does not match the master branch on github.  All the bad-continuation stuff is still included in my local copy.

**Comment by danrneal** [comment]
Actually the problem is with the tag for 2.5.3 in git is exactly the same as 2.5.2 which is exactly the same as 2.5.1 which is exactly the same as 2.5.0.  Essentially it's as if no work after 2.5.0 has actually been released.

**Comment by danrneal** [comment]
@PCManticore It looks as if you are doing the tagging, you may want to take a look at this.  Sorry for the multiple messages/pings.

**Comment by dbaty** [comment]
pylint 2.6.0 has been released since you opened this issue. I cannot reproduce the bug with this version. (Note that I needed to add `--enable=bad-continuation` to reproduce with pylint 2.5.*)

I don't think that versions 2.5.x should be replaced on PyPI. So we should probably close this,shall we?

**Comment by Pierre-Sassoulas** [comment]
Good point @dbaty, thank you for trying to reproduce !
